data/sql/updates/pending_db_world/fix_issue18729.sql

```diff
@@ -0,0 +1,7 @@
+
+DELETE FROM `creature_template_spell` WHERE `CreatureID` = 27354;
+INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
+    (27354, 0, 48548, 12340),
+    (27354, 1, 48558, 12340),
+    (27354, 2, 48604, 12340),
+    (27354, 4, 48610, 12340);
```

Comment:
```suggestion
INSERT INTO `creature_template_spell` (`CreatureID`, `Index`, `Spell`, `VerifiedBuild`) VALUES
(27354, 0, 48548, 12340),
(27354, 1, 48558, 12340),
(27354, 2, 48604, 12340),
(27354, 4, 48610, 12340);
```

Reply:
These entries need their `VerifiedBuild` set to 53788 as well. (Grabbed the first build I could find in which I have a sniff of this creature being charmed)